lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8303,6 +8303,17 @@ bool SwiftASTContext::GetImplicitImports(
   return true;
 }
 
+void SwiftASTContext::LoadStringProcessingModule(TargetSP target,
+                                                 ProcessSP process) {
+  llvm::call_once(m_load_string_processing_lib_once, [&]() {
```

Comment:
IIUC, the call_once is redundant?

Reply:
On further inspection I believe it's redundant as `LoadOneModule` checks if the module is cached before loading it.